theming/_variables.scss

```
@@ -175,13 +175,13 @@ footer {
 }
 
 a {
-  color: rgba(0, 0, 0, 0.6) !important;
+  color: rgb(0, 68, 170) !important;
   background-color: transparent;
   text-decoration: none;
 }
 
-.aboutPage  {
-  color: rgb(0, 68, 170) !important;
+.footerLinks  {
+  color: rgba(0, 0, 0, 0.6) !important;
   background-color: transparent;
   text-decoration: none;
 }
```

```suggestion
.footer-link {
  color: rgba(0, 0, 0, 0.6) !important;
  background-color: transparent;
  text-decoration: none;
}
```
I forgot to mention earlier but since we already have a .footer-link:hover class/selector then we can just go ahead and tag on the gray color for footer links onto that class